src/Tizen.Maps/Tizen.Maps/MapView.cs

```diff
@@ -755,6 +755,8 @@ private void UnsetViewReadyEventCallback()
             {
                 handle.UnsetEventCb(Interop.ViewEventType.Ready);
                 _viewReadyEventCallback = null;
+
+
                 Log.Info(string.Format("ViewReady event callback is unset"));
             }
```

Comment:
The behavior of 'string.Format(string, params object[])' could vary based on the current user's locale settings. Replace this call in 'MapView.UnsetViewReadyEventCallback()' with a call to 'string.Format(IFormatProvider, string, params object[])'.